### PR TITLE
created the skeleton for mvc routes

### PR DESCRIPTION
**Pull request by donnatopia**
- I created the files "controllers," "models," and "routes" in the server folder to organize the servers to the MVC structure
- I created the connection to the database in the db folder
- I also adjusted the postgres database to include related products table and to use the \c connection

### CODE REVIEW

File: server/db/connection.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { Client } = require('pg');
+const dotenv = require('dotenv');
+
+const client = new Client({
+  user: process.env.USER,
+  port: process.env.PORT,
```
Comment (wooseok0717) [inline server/db/connection.js:6]: If I am not wrong I think .env port is 3000 and db stays in different port.

File: server/index.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 const app = express();
 
-// ADD APP-WIDE MIDDLEWARE
+// middleware
 app.use(morgan('dev'));
 app.use(cors());
 app.use(express.json());
 
 app.use(express.static(path.join(__dirname, '../test')));
+app.use('/products', router);
```
Comment (wooseok0717) [inline server/index.js:17]: did you forget to require router? or am I not seeing it?

File: server/controllers.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+const models = require('./models.js');
+
+module.exports = {
+  getProducts: (req, res) => {
+    models.getProducts(req)
+      .then(({ data }) => {
+        res.status(200);
+        res.end(JSON.stringify(data));
```
Comment (wooseok0717) [inline server/controllers.js:8]: using res.json would be much clean :)

File: server/db/products.sql [hunk 4]
```diff
@@ -1,7 +1,7 @@
 DROP DATABASE IF EXISTS products;
 CREATE DATABASE products;
 
-USE products;
+\c products;
 
 CREATE TABLE products {
   id SERIAL PRIMARY KEY,
```
Comment (wooseok0717) [inline server/db/products.sql]: just a question what happens if product have their id numbers in CSV?

File: server/index.js (hunk 2 above)
Comment (donnatopia) [reply server/index.js:17]: yes i did! good catch!

File: server/controllers.js (hunk 3 above)
Comment (donnatopia) [reply server/controllers.js:8]: will do in later iterations

File: server/db/connection.js (hunk 1 above)
Comment (donnatopia) [reply server/db/connection.js:6]: will change

**Pull request merged**